packages/core/admin/admin/src/content-manager/components/RelationInput/RelationInput.js

```diff
@@ -312,14 +371,25 @@ const RelationInput = ({
         }
       >
         <RelationList overflow={overflow}>
+          <VisuallyHidden id={`${name}-item-instructions`}>
+            {formatMessage({
```

Comment:
Can we please pass that through as a prop? If we decide to move the component into the DS we shouldn't have a dependency on react-intl.